translator/context/context.go

```diff
@@ -127,8 +127,17 @@ func (ctx *Context) SetMode(mode string) {
 	case config.ModeWithIRSA:
 		ctx.mode = config.ModeWithIRSA
 		ctx.shortMode = config.ShortModeWithIRSA
+	case config.ModeEKS:
+		ctx.mode = config.ModeEKS
+		ctx.shortMode = config.ShortModeEKS
+	case config.ModeK8sEC2:
+		ctx.mode = config.ModeK8sEC2
+		ctx.shortMode = config.ShortModeK8sEC2
+	case config.ModeK8sOnPrem:
+		ctx.mode = config.ModeK8sOnPrem
+		ctx.shortMode = config.ShortModeK8sOnPrem
 	default:
-		log.Panicf("Invalid mode %s. Valid mode values are %s, %s, %s and %s.", mode, config.ModeEC2, config.ModeOnPrem, config.ModeOnPremise, config.ModeWithIRSA)
+		log.Panicf("Invalid mode %s. Valid mode values are %s, %s, %s, %s, %s, %s, and %s.", mode, config.ModeEC2, config.ModeOnPrem, config.ModeOnPremise, config.ModeWithIRSA, config.ModeEKS, config.ModeK8sEC2, config.ModeK8sOnPrem)
```

Comment:
I don't think we should be listing out all the modes. Don't think this can be reached, but if it were logged, it would give the wrong impression to that these new modes can be set with `-m`.